### PR TITLE
Using logger instead of print

### PR DESCRIPTION
**Pull request by Guillem96**
*Description of changes:*

Instead of using `print` function use the `logger`.

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I certify that the changes I am introducing will be backword compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [x] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by vikas0203 — COMMENTED** [review]
/bot run all

**Review by vikas0203 — APPROVED** [review]

File: src/sagemaker/processing.py [hunk 1]
```diff
@@ -787,10 +787,9 @@ def start_new(cls, processor, inputs, outputs, experiment_config):
         process_args = cls._get_process_args(processor, inputs, outputs, experiment_config)
 
         # Print the job name and the user's inputs and outputs as lists of dictionaries.
```
Comment (mufaddal-rohawala) [inline src/sagemaker/processing.py:789]: Can we update the comment as well?

**Review by mufaddal-rohawala — CHANGES_REQUESTED** [review]
tests are failing can you check units?

**Comment by mufaddal-rohawala** [comment]
Closing PR as there are no updates for a long time.

**Pull request closed without merging**